utils/_context/containers.py

```diff
@@ -718,10 +718,6 @@ def __init__(
             base_environment["DD_IAST_DEDUPLICATION_ENABLED"] = "false"
 
         if tracer_sampling_rate:
-            if context.library == "python":
-                # python3.x dropped support for DD_TRACE_SAMPLE_RATE
-                base_environment["DD_TRACE_SAMPLING_RULES"] = str([{"sample_rate": tracer_sampling_rate}])
```

Comment:
envvars set in the `base_environment` are overwritten. I need to define this configuration in the scenario,